test/ui/browser/test-browser-util.js

```diff
@@ -18,7 +23,12 @@ describe('utils', function() {
 
     it('should append protocol onto URIs', function() {
       const CHANGED_URIS = [
-        ['example.com', 'http://example.com'],
+        ['example.com', 'http://example.com/'],
+        ['example.net', 'http://example.net/'],
+        ['foo.bar', 'http://foo.bar/'],
+        ['blerg.co.uk', 'http://blerg.co.uk/'],
+        ['blach.com', 'http://blach.com/'],
+        ['www.blah.com', 'http://www.blah.com/'],
```

Comment:
We should test leading with a space, like '   mozilla.org' -> 'http://mozilla.org'